woocommerce/woocommerce-android
# [Woo POS] Connect to card reader when clicking on the connection status

Closes: #11664, #11668


### Description

This PR displays the current reader connection status on the bottom toolbar and allows connecting to a card reader by clicking on the card reader status.

### Testing information

1. Click on the "Reader disconnected" button and ensure the reader connection flow runs successfully
2. Ensure reader status is reflected in the bottom toolbar (disconnected / connecting / connected)

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4527432/3d4a0e79-a2a2-4382-b878-6094e78678d8
